changelog

```diff
@@ -1,3 +1,243 @@
+Version 0.9.13:
+  * Big changes:
+    * Added over 30 new spaceport missions to Coalition space. (@Arachi-Lover)
+    * Added over 25 new missions to Remnant space, as well as a new region of the Ember Waste. (@Zitchas)
+    * Added 15 new archaeology missions with Albert Foster to investigate the Sheragi ruins, including the discovery of ancient Sheragi technology. (@Karirawri)
+    * The Ember Waste has become a more hostile place with the introduction of system hazards. Be prepared the next time you visit. (@Zitchas)
+    * Outfits can now be stored on planets by pressing U or R on installed or in cargo outfits in the outfitter. (@petervdmeer)
+    * Completely redid and improved the graphics for most human outfit images. (@beccabunny)
```

Comment:
```suggestion
    * Completely redid and improved the graphics for most human and korath outfit images. (@beccabunny)
```
Since it is more correct

Reply:
The Korath image updates are mentioned down in the graphics section. I decided to only mention the human outfits here as a big change since those were the oldest outfit images and all a cohesive group, while "Korath outfits" is a different group.